src/Persistence/MassTransit.Azure.Table/Contexts/AzureTableDatabaseContext.cs

```diff
@@ -0,0 +1,105 @@
+namespace MassTransit.Azure.Table
```

Comment:
namespace?

Reply:
What's the change you're looking for here?